test/test_sparse.py

```diff
@@ -2363,6 +2363,304 @@ def test_sparse_to_numpy(self):
         t = torch.sparse_coo_tensor(torch.tensor(([0, 0], [2, 0])), torch.tensor([1, 4]))
         self.assertRaises(TypeError, lambda: t.numpy())
 
+    def test_softmax(self):
+        if self.device != 'cpu':
+            self.skipTest('sparse softmax not available on %s' % (self.device))
+
+        import torch.nn.functional as F
+
+        def to_dense(sparse, fill_value=None):
+            """
+            Return dense tensor from a sparse tensor using given fill value.
+            """
+            if isinstance(fill_value, str):
```

Comment:
lolwut

Reply:
yeah, that was used for `fill_value=float("-inf")`, now removed.